buckets.tf

```diff
@@ -158,6 +158,15 @@ resource "google_storage_bucket_iam_policy" "tenant_output_bucket" {
   policy_data = data.google_iam_policy.tenant_output_bucket.policy_data
 }
 
+# Enable object events to required PubSub topics
+resource "google_storage_notification" "output_bucket_notificaions" {
+  for_each       = var.output_bucket_storage_notification
+  bucket         = google_storage_bucket.tenant_output_bucket.name
+  payload_format = "JSON_API_V1"
+  topic          = each.value
```

Comment:
part of the discussion in the slack thread:
https://liveramp.slack.com/archives/C03D27C80HM/p1709105157624179

---
If we make this property configurable and a user can input any topic here, how do we know we will have access to publish? If this is meant to reside on data-plane side, should we then provide a topic managed by the module following the same patter we have in control-plane:
```
public.identity-engine.notification-channel
```

and make this publish to that topic only?